### DISCUSSION TITLE
Stats Map for fantasy basketball

### DISCUSSION THREAD

**Discussion by jwill24** [cwendt94/espn-api#177] [ESPN API] [answered]
Hey, I noticed a small error in the stats map.

In espn_api/basketball/constant.py, in STATS_MAP, 41 = GS and 42 = GP.

Thank you for this code. It is incredibly helpful!!

**Comment by cwendt94** [comment, marked as answer]
Thanks for pointing this out! I will update and put it in the next version!

I am glad this package has been helpful!

**Reply by cwendt94** [reply]
Also if you know what some of the other unfilled stat numbers mean, feel free to post it here!
